cyberdrop_dl/utils/args/config_definitions.py

```diff
@@ -125,6 +125,11 @@
         "sorted_other": "{sort_dir}/{base_dir}/Other/{filename}{ext}",
         "sorted_video": "{sort_dir}/{base_dir}/Videos/{filename}{ext}",
     },
+    "Browser_Cookies": {
+        "browser": "Chrome",
+        "auto_import": False,
+        "sites": None
+    },
```

Comment:
NIT: I think `sites` is redundant. `auto_import` could have the sites as a list. An empty list would mean `auto_import` disabled.

Reply:
Well the idea was to allow the user to decide what to import

Reply:
That makes sense! Yeah, an empty list as `Every site` is probably a better UX that manually entering all the sites. Good call